apps/test-can/include/CANConfig.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "CANInterface.h"
+#include "CANMsg.h"
+#include "hw_bridge.h"
+
+static mbed_error_status_t handle_test_msg_one_shot(CANMsg& msg);
+
+namespace CANConfig {
+
+using namespace HWBRIDGE;
+
+static CANMsgMap rxStreamedMsgMap = {
+    // Msg 1
+    {COMMON_TEST_MESSAGE2,
+     {
+         {COMMON_TEST_SIGNAL2, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL2_TEST_VALUE_2_CHOICE},
+         {COMMON_TEST_SIGNAL3, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL3_SNA_CHOICE},
+     }},
+
+    // Msg 2
+};
+
+static CANMsgMap txStreamedMsgMap = {
+    // Msg 1
+    {COMMON_TEST_MESSAGE2,
+     {
+         {COMMON_TEST_SIGNAL2, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL2_TEST_VALUE_2_CHOICE},
+         {COMMON_TEST_SIGNAL3, UWRT_MARS_ROVER_CAN_COMMON_TEST_MESSAGE2_COMMON_TEST_SIGNAL3_SNA_CHOICE},
+     }},
+
+    // Msg 2
+};
```

Comment:
For each target, we'll need to configure a RX msg map and a TX msg map that lists all the messages that target is receiving / transmitting (streamed messages only!). Msg map structure is:
```
{
msg1Name : {signal1Name : signal1Value, signal2Name : signal2Value, ...},
msg2Name : {signal1Name : signal1Value, signal2Name : signal2Value, ...},
...
}
```